src/components/articles-components/VoteButtons.jsx

```diff
@@ -0,0 +1,61 @@
+import { useState, useEffect } from "react";
+import { updateArticleVotes } from "../../functions/api";
+
+const VoteButtons = ({ articleId, initialVotes }) => {
```

Comment:
It is a React anti-pattern to initialise state with a value from props and will cause unexpected and difficult bugs